### PR TITLE
Update views and update RS server description

### PR DESCRIPTION
**Pull request by pcuq-ads**
Update of architecture diagrams with last version.
Split RS-Server and RS-Python (other PULL request will complete this task on repository rs-documentation).

### CODE REVIEW

**Review by Padeanu — APPROVED** [review]

File: docs/doc/index.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 
-The toolbox provides REST endpoints that enable users to:
+To achieve this, RS-Server exposes REST endpoints that allow users to:
+* Search CADU chunks from CADIP stations
+* Stage CADU chunks from CADIP stations
+* Search auxilliary data from AUXIP station
+* Stage auxilliary data from AUXIP station
+* ALL endpoints from [SpatioTemporal Asset Catalog API](https://stacspec.org/) 
+
+Please note that the STAC catalog will also embedds STAC extension to support Sentinel product format from [EOPF-CPM](https://cpm.pages.eopf.copernicus.eu/eopf-cpm/main/index.html).
```
Comment (agrosu1978) [inline docs/doc/index.md:16]: * will also embedds -> it's future, the verb should be used with the base form of the verb -> will also embed

File: docs/doc/index.md [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 
 ## Features
 
-The toolbox provides REST endpoints that enable users to:
+To achieve this, RS-Server exposes REST endpoints that allow users to:
+* Search CADU chunks from CADIP stations
+* Stage CADU chunks from CADIP stations
+* Search auxilliary data from AUXIP station
+* Stage auxilliary data from AUXIP station
+* ALL endpoints from [SpatioTemporal Asset Catalog API](https://stacspec.org/)
```
Comment (agrosu1978) [inline docs/doc/index.md:14]: The last statement is not linked with the primary phrase:
```To achieve this, RS-Server exposes REST endpoints that allow users to: ALL endpoints from.......```
Suggestion:
```To achieve this, RS-Server exposes REST endpoints that allow users to: connect to ALL endpoints from....... ```

**Review by agrosu1978 — CHANGES_REQUESTED** [review]

File: docs/doc/index.md (hunk 1 above)
Comment (pcuq-ads) [reply docs/doc/index.md:16]: Fine for me. Your english is better than mine. So fill free to fix any wording.

File: docs/doc/index.md (hunk 2 above)
Comment (pcuq-ads) [reply docs/doc/index.md:14]: OK.

**Review by agrosu1978 — APPROVED** [review]

**Pull request merged**